client/rpc-api/src/system/mod.rs

```diff
@@ -65,6 +67,21 @@ pub trait SystemApi<Hash, Number> {
 	#[rpc(name = "system_networkState", returns = "jsonrpc_core::Value")]
 	fn system_network_state(&self) -> Receiver<jsonrpc_core::Value>;
 
+	/// Adds a reserved peer. Returns the empty string or an error. The string
```

Comment:
returns `()`?

Reply:
As I understand it, this will serialize to the empty string over RPC.

Reply:
Ok, I see

Reply:
Actually `()` is serialized as `null` afair. Might be better to return a `bool` maybe? There was some discussion around `author_insertKey` returning `null`, that it's not an expected result from a succesfull operation.